freeipa/src/main/java/com/sequenceiq/freeipa/flow/freeipa/verticalscale/event/FreeIPAVerticalScaleEvent.java

```diff
@@ -0,0 +1,29 @@
+package com.sequenceiq.freeipa.flow.freeipa.verticalscale.event;
+
+import com.sequenceiq.flow.core.FlowEvent;
+import com.sequenceiq.flow.event.EventSelectorUtil;
+
+public enum FreeIPAVerticalScaleEvent implements FlowEvent {
+    STACK_VERTICALSCALE_EVENT("STACK_VERTICAL_SCALE_TRIGGER_EVENT"),
+    STACK_VERTICALSCALE_FINISHED_EVENT(EventSelectorUtil.selector(FreeIPAVerticalScaleResult.class)),
+    STACK_VERTICALSCALE_FINISHED_FAILURE_EVENT(EventSelectorUtil.failureSelector(FreeIPAVerticalScaleRequest.class)),
+
+    STACK_VERTICALSCALE_FINALIZED_EVENT,
+    STACK_VERTICALSCALE_FAILURE_EVENT,
```

Comment:
this is used only in flowconfig. maybe you would like to add the `(EventselectorUtil.selector(FreeIPAVerticalScaleFailureEvent.class))` for this